pkgs/development/typst-packages/cetz.nix

```
@@ -0,0 +1,35 @@
+{
+  buildTypstPackage,
+  fetchFromGitHub,
+  just,
+  file,
+  perl,
+}:
+
+buildTypstPackage rec {
+  pname = "cetz";
+  version = "0.2.2";
+
+  src = fetchFromGitHub {
+    owner = "cetz-package";
+    repo = "cetz";
+    rev = "v${version}";
+    hash = "sha256-5+Np64+0Ca8yL7D1CCUewF03Wh6BM2bV1BQNUA+bW74=";
+    deepClone = true;
+  };
+
+  nativeBuildInputs = [
+    just
+    file
+    perl
+  ];
+
+  buildPhase = ''
+    patchShebangs scripts/package
+    HOME=$(pwd) just install
+  '';
```

Overriding entire phases seem really weird to me, especially since there are hooks for `just` that run such commands.

```suggestion
  prePatch = ''
    patchShebangs scripts/package
  '';
  
  postPatch = ''
    export HOME=$(mktemp -d)
  '';
```

---

Thanks for your review! The package builder now simply moves all files in the home directory to the desired output directory. Any typst package that requires a build phase to generate typst scripts (e.g., cetz) can override `buildPhase` (as the package builder does nothing in `buildPhase`) and should make sure the resulting typst scripts locates in the home directory (using `preInstall` hook).